File: JAVA PROGRAMS/RecursivePowerMethod.java
import java.util.Scanner;

public class RecursivePowerMethod {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int base, exponent;

        System.out.print("Enter the number to be raised: ");
        base = input.nextInt();

        System.out.print("Enter the exponent: ");
        exponent = input.nextInt();

        System.out.println(base + " raised to " + exponent + " is equal to " + power(base, exponent));
        System.out.println("\t" + base + "^" + exponent + " = " + power(base, exponent));
        input.close();
    }
    public static int power(int base, int exponent) {
        if (exponent == 0) return 1;
        else return base * power(base, exponent-1);
    }
}

File: PYTHON PROGRAMS/RecursivePowerMethod.py
def main():
    base = int(input("Enter the number to be raised: "))
    exponent = int(input("Enter the exponent: "))

    print(str(base) + " raised to " + str(exponent) + " is equal to " + str(power(base, exponent)))
    print("\t" + str(base) + "^" + str(exponent) + " = " + str(power(base, exponent)))

def power(base, exponent):
    if exponent == 0: return 1
    else: return base * power(base, exponent-1)
    
main()